CUDADataFormats/BeamSpot/interface/BeamSpotCUDA.h

```diff
@@ -4,15 +4,19 @@
 #include <cuda_runtime.h>
 
 #include "DataFormats/BeamSpot/interface/BeamSpotPOD.h"
-#include "HeterogeneousCore/CUDAUtilities/interface/device_unique_ptr.h"
+#include "HeterogeneousCore/CUDAUtilities/interface/cudaMemoryPool.h"
 
 class BeamSpotCUDA {
 public:
+  using Buffer = memoryPool::Buffer<BeamSpotPOD>;
```

Comment:
Hi @VinInn isnt this technically a namespace? According to rule 2.7 those should start with a lowercase letter

Reply:
No, it's a type alias: https://en.cppreference.com/w/cpp/language/type_alias .
So I guess it's OK for it to follow the same rule as for classes.